Gaijinizer patch/Map036.txt

```diff
@@ -25,15 +27,15 @@ What……!!What……!!Yu, a dream .........?#MTLed
 アイアンメイデンに負けて…
 > CONTEXT: Map036/events/13/pages/1/18/Dialogue/kazuya
 > CONTEXT: Map036/events/13/pages/2/17/Dialogue/kazuya
-That's right ...! I am sure ...
-Lose to Iron Maiden ...#MTLed
+O- oh yeah...! Last thing I remember was losing to
+Iron Maiden...#Ron
```

Comment:
I'd move "to" to the next line.

In general, try to make sure that we don't put more then 50 characters in a line. The text box can support a few more, but 50 gives the proper padding.